test/core.vwtest.json

```diff
@@ -2474,13 +2474,13 @@
   },
   {
     "id": 208,
-    "desc": "Enable chain hash option for text example",
+    "desc": "Use chain hashing for text example",
     "diff_files": {
       "stderr": "test-sets/ref/chain_hash_text_result.stderr",
       "stdout": "test-sets/ref/chain_hash_text_result.stdout",
       "chain_hash_text_result.cmp": "test-sets/ref/chain_hash_text_result.cmp"
     },
-    "bash_command": "{VW} --audit -d train-sets/chain_hash_text_test.dat --invert_hash chain_hash_text_result.cmp --chain_hash --predict_only_model &&  tail -n +2 chain_hash_text_result.cmp > chain_hash_text_result.cmp.new &&  rm chain_hash_text_result.cmp &&  mv chain_hash_text_result.cmp.new chain_hash_text_result.cmp",
+    "bash_command": "{VW} --audit -d train-sets/chain_hash_text_test.dat --invert_hash chain_hash_text_result.cmp --predict_only_model &&  tail -n +2 chain_hash_text_result.cmp > chain_hash_text_result.cmp.new &&  rm chain_hash_text_result.cmp &&  mv chain_hash_text_result.cmp.new chain_hash_text_result.cmp",
```

Comment:
why did we remove `chain_hash` arg here?

Reply:
`--chain_hash` does nothing when `--json` or `--dsjson` isn't being used. It is automatic for text. I removed it since redundant options in the tests are confusing.